docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -87,4 +76,8 @@ player supports toggling between play and pause, but does not support seeking to
 a specific point in the stream, rewinding, or restarting playback when the end
 of the stream is reached. To replay a session, refresh the page.
 
+## Log rotation
 
+Be aware, Desktop Access session recording comprises of raw PNG-based files.
+Consider the frequency and duration of Desktop Access sessions and rotate logs
```

Comment:
Does "Consider the frequency and duration of Desktop Access sessions" mean that you might want to limit Desktop Access sessions, or that you should be tracking the length of these sessions so you can size your disk volumes correctly?

Reply:
As mentioned offline, there's no need to consider manually rotating logs in most deployments.

Reply:
Removed this sentence.